src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToItemOrderConfirmationMapper.java

```diff
@@ -43,42 +44,65 @@
     @Mapping(source = "reference", target="orderReferenceNumber")
     @Mapping(source = "orderedBy.email", target="emailAddress")
     @Mapping(source = "totalOrderCost", target="totalFee")
-    public abstract CertifiedCopyOrderConfirmation orderToConfirmation(OrderData order);
+    public abstract ItemOrderConfirmation orderToConfirmation(OrderData order);
 
     @AfterMapping
     public void mapCertifiedCopyItems(final OrderData order,
-                                       final @MappingTarget CertifiedCopyOrderConfirmation confirmation) {
+                                       final @MappingTarget ItemOrderConfirmation confirmation) {
         final Item item = order.getItems().get(0);
-        final String timescale = item.getItemOptions().getDeliveryTimescale().toString();
 
         confirmation.setCompanyName(item.getCompanyName());
         confirmation.setCompanyNumber(item.getCompanyNumber());
-        String deliveryMethod = String.format("%s delivery (aim to dispatch within 4 working days)", toSentenceCase(timescale));
-        confirmation.setDeliveryMethod(deliveryMethod);
+
+        if (item.getKind().equals("item#certified-copy")) {
+            final String timescale = item.getItemOptions().getDeliveryTimescale().toString();
+            String deliveryMethod = String.format("%s delivery (aim to dispatch within 4 working days)", toSentenceCase(timescale));
+            confirmation.setDeliveryMethod(deliveryMethod);
+            confirmation.setItemDetails(collateItemDetailsForCertifiedCopy(item));
+        }
+        else {
+            confirmation.setItemDetails(collateItemDetailsForMissingImageDelivery(item));
```

Comment:
The method name says certified copies, but it also handles mid? Is certificates handled elsewhere?

Reply:
Yes it is a bit confusing isn't it? Perhaps we can rename this method to reflect the more generic repurposing it was recently given on `master`.

The mapping for certificates is carried out by the `OrderDataToCertificateOrderConfirmationMapper`.

Reply:
Yeh cool, go for it.